fastf1/ergast/interface.py

```diff
@@ -478,6 +481,9 @@ def _build_url(
         if endpoint is not None:
             selectors.append(f"/{endpoint}")
 
+        if standings_position is not None:
```

Comment:
This doesn't seem right. Simply appending the standings position at the end of the API path can't be correct. 
I think this is already properly handled on master after merging your other PR, no?

Reply:
When working these three extra functions
- `get_race_results()`
- `get_qualifying_results()`
- `get_sprint_results()`
We need to add the standing position **after** the endpoint.

This is exactly the way `get_driver_standings()` and `get_constructor_standings()` behave.
We modify the endpoint by appending the standing_position.
The standing position always needs to come after the endpoint, whether modified, like the standing functions, or with the default endpoint.

Alternatively we can consider adding final else statement:
 ```
if standings_position is not None:
    if endpoint == 'driverStandings':
        endpoint = f"driverStandings/{standings_position}"
    elif endpoint == 'constructorStandings':
        endpoint = f"constructorStandings/{standings_position}"
    else:
        endpoint = f"{endpoint}/{standings_position}"
```

In the previous PR there was nothing directly expanding this functionality as the three functions do not have the standings_position parameter

Reply:
Ok, right, this might work, but what actually changes here?

The code was 
```
selectors.append(f"/constructorStandings/{standings_position}")
```
it now is
```
selectors.append(f"/constructorStandings")
# ...
selectors.append(f"/{standings_position}")
```
with the second line being inside a conditional that is always true if the conditional for the first line was true. And additionally, with some code in between that could theoretically insert stuff into the selectors as well, but that won't be encountered in reality.

So in the end nothing changes but the code flow becomes much less obvious. What is the point of this? Am I missing something?

Reply:
For constructorStandings and driverStandings nothing would change, seen in the alternative I provided earlier.

The below code change and along with appending the parameter, standings_position, to the 3 functions mentioned will expand the functionality to get race, sprint, and qualifying results by a specific position.
```
if standings_position is not None:
    ...
    else:
        endpoint = f"{endpoint}/{standings_position}"
```
Here is an example:
You want to find **only** the first place finishers of 2018. 
By adding the standings_position to the endpoint, `{BASE_URL}/2018/results/1`, we get the desired results.

Reply:
Right, now I understand. 

There already was the properly named ``results_position`` argument for that. But it was in fact neither used nor properly implemented. I just fixed that to do what you intended to do with the ``standings_position`` argument and I fixed the merge conflicts.

Reply:
I really like what you did with that, makes perfect sense to use results_position over standings_position. Glad this PR also worked out.